docs/model_zoo.md

```diff
@@ -193,6 +193,10 @@ We also train [Faster R-CNN](https://github.com/open-mmlab/mmdetection/blob/mast
 
 ## Speed benchmark
 
+### Training Speed benchmark
+
+We provide [analyze_logs.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analyze_logs.py) to get the overall training time and you can find an example in [Log Analysis](https://mmdetection.readthedocs.io/en/latest/useful_tools.html#log-analysis).
```

Comment:
get the overall training time -> average time of iteration in training. You can find examples in xxx